test/ClangImporter/Inputs/typedef-with-generic-param.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+@interface Foo <__covariant T>
+typedef void (^CompletionBlock)(T _Nullable result, NSError *_Nullable error);
```

Comment:
Why are we unable to import this typedef though? It should be no different than the Swift code and the ClangImporter should be able to construct it without problems:

```
class Foo<T> {
   typealias CompletionBlock = () throws -> T
}
```